tests/conftest.py

```diff
@@ -34,8 +34,8 @@ def depcode_serpent(cwd):
 
 
 @pytest.fixture(scope='session')
-def depcode_openmc(cwd):
-    """DepcodeOpenMC object for unit tests"""
+def openmc_depcode(cwd):
+    """OpenMCDepcode object for unit tests"""
     saltproc_input = (cwd / 'openmc_data' / 'tap_input.json').as_posix()
     _, _, _, object_input = read_main_input(saltproc_input)
```

Comment:
I don't love this... I get that the first several variables are unused -- but I think they should still have proper names. I don't think there are any performance benefits to this either. This is just a preference though, so if you have a compelling reason to use underscores go ahead, I suppose.

Reply:
My perspective is that they are not relevant to anything else in the function, and giving them a null-like name indicates that. Maybe @munkm has some perspective on this?